### ISSUE TITLE
using for downstream 

### ISSUE THREAD

**Issue by niloole** [SIAT-code/MASSA#5]
Hello
I want to do a downstream task. but I think the pre-trained model parameter does not exist.

```
  ## loading pretrained model
    pretrained_dict = torch.load("pretrained/epoch150-model-2022-05-25-12:57:55/model--2022-05-25-12:57:55.pth", map_location=torch.device('cpu'))
```
   
I could not find this file: pretrained/epoch150-model-2022-05-25-12:57:55/model--2022-05-25-12:57:55.pth
How can I use your method to extract representation for protein to do my downstream task by adding an FC layer and sigmoid?
